test/unit/autocomplete.ts

```diff
@@ -215,6 +215,19 @@ suite('Autocomplete', ({ expect, spy }) => {
       expect(set).to.be.calledWith({ suggestions, navigations, categoryValues, selected: -1 });
       expect(setActivation).to.be.calledWith(targets, selected, false);
     });
+
+    it('should inactivate sayt when there are no suggestions', () => {
+      const suggestions = [];
+      const navigations = [];
+      const categoryValues = [];
+      const emit = autocomplete.flux.emit = spy();
+
+      autocomplete.set = () => null;
+
+      autocomplete.updateSuggestions(<any>{ suggestions, navigations, category: { values: categoryValues } });
+
+      expect(emit).to.be.calledOnce;
```

Comment:
here we should be expecting that emit is `calledWith('sayt:hide')`, not just that it's called